tests/symbolicator/test_payload_full.py

```diff
@@ -104,7 +104,8 @@ def test_real_resolving(self):
         assert response.status_code == 201, response.content
         assert len(response.data) == 1
 
-        event = self.post_and_retrieve_event(REAL_RESOLVING_EVENT_DATA)
+        with self.feature({"organizations:images-loaded-v2": False}):
```

Comment:
Since we're GA-ing this (basically, making a default), should we also update the tests?
Otherwise we'll be testing not what we're using.

Reply:
(unless we have tests for both states)

Reply:
The file also has a `test_resolving_with_candidates_sentry_source` which explicitly checks this feature.  I was considering updating the snapshots instead of this, but this makes the snapshots less brittle while the feature is still tested.  Which I think was the reasoning we chose when we implemented this the first time round.

Reply:
Having the test suite independent of the feature flag also makes it easier to toggle the flag.  There could be an argument that we shouldn't do this in the same PR to make this easier even.  I'll leave that choice up to you as you're most likely to be flipping it back if needed.

Reply:
> The file also has a test_resolving_with_candidates_sentry_source which explicitly checks this feature.

That's good, but the test coverage for code that matters (i.e. that actually runs in production and on-prem by default) actually dropped, no? 1 test versus N tests that we had with the features disabled.

Reply:
When we implemented this we decided there was sufficient test coverage for the feature in symbolicator itself, which tests this more extensively.  The main functionality of the tests here is to ensure that sentry actually switches on this functionality and merges it into the event.  The actual contents of the extra stuff is tested in symbolicator.